crates/common/src/job_witness.rs

```diff
@@ -1,5 +1,5 @@
 use crate::hash;
-use cairo_felt::Felt252;
+use cairo_vm::Felt252;
```

Comment:
what if we use `starknet::FieldElement` type? think better to have unified type either Felt252 or FieldElement